Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Cortex_Endpoint_Enrichment.yml

```diff
@@ -0,0 +1,1431 @@
+id: Cortex ASM - Cortex Endpoint Enrichment
+inputs:
+- description: IP address of service
+  key: RemoteIP
+  playbookInputQuery:
+  required: false
+  value: {}
+name: Cortex ASM - Cortex Endpoint Enrichment
+outputs: []
+starttaskid: "0"
+tasks:
+  "0":
+    continueonerrortype: ""
+    id: "0"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "9"
+      - "44"
+    note: false
+    quietmode: 0
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: ""
+      id: 0238da21-5885-4f70-804b-f6eeac7dbcd7
+      iscommand: false
+      name: ""
+      version: -1
+      description: ''
+    taskid: 0238da21-5885-4f70-804b-f6eeac7dbcd7
+    timertriggers: []
+    type: start
+    view: |-
+      {
+        "position": {
+          "x": 670,
+          "y": -40
+        }
+      }
+  "6":
+    continueonerror: true
+    continueonerrortype: ""
+    id: "6"
+    ignoreworker: false
+    isautoswitchedtoquietmode: false
+    isoversize: false
+    nexttasks:
+      '#none#':
+      - "10"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      public_ip_list:
+        complex:
+          root: inputs.RemoteIP
+    separatecontext: false
+    skipunavailable: true
+    task:
+      brand: ""
+      description: Gets a list of endpoints, according to the passed filters. If there are no filters, all endpoints are returned. Filtering by multiple fields will be concatenated using AND condition (OR is not supported). Maximum result set size is 100. Offset is the zero-based number of endpoint from the start of the result set (start by counting from 0).
```

Comment:
```suggestion
      description: Gets a list of endpoints, according to the passed filters. If there are no filters, all endpoints are returned. Filtering by multiple fields will be concatenated using the AND condition (OR is not supported). Maximum result set size is 100. Offset is the zero-based number of endpoints from the start of the result set (start by counting from 0).
```

Reply:
will change but better (more permanent fix) would be made in https://github.com/demisto/content/blob/ec57869baf127c2b3d17d18c3019a3545ffc2845/Packs/Core/Integrations/CortexCoreIR/CortexCoreIR.yml#L315